www/api/modules/payments/infraestructure/controllers/subscription_type/subscription_type_http_controller.py

```diff
@@ -27,3 +27,34 @@ def all():
     subscription_types = subscription_type_app.get_all()
 
     return Response(json.dumps(subscription_types), status=200)
+
+
+@subscription_type_api.route('', methods=['POST'])
+@api_authorizer([UserType.ADMIN])
+def create(user):
+    data = request.get_json()
+
+    errors, subscription_type = subscription_type_app.create(user, data)
+
+    if errors:
+        return Response(json.dumps({"errors": errors}), status=404)
+
+    subscription_type_dict = as_dict(subscription_type)
+
+    return Response(json.dumps(
+        {
+            'id': subscription_type_dict.get('id'),
+            'name': subscription_type_dict.get('name'),
+            'price': subscription_type_dict.get('price'),
+            'currency': subscription_type_dict.get('currency'),
+            'payment_cycle': subscription_type_dict.get('payment_cycle'),
+            'is_active': subscription_type_dict.get('is_active')
+        }, default=str
+    ), status=201)
```

Comment:
Why not just use as_json_dumps

```suggestion
    return Response(as_json_dumps(subscription_type), status=201)
```

Reply:
because its more clean this way